props/saber_fett263_buttons.h

```diff
@@ -0,0 +1,307 @@
+        // Avoid the base and the very tip.
+        // TODO: Make blast only appear on one blade!
+        swing_blast_ = false;
+        SaberBase::DoBlast();
+        return true;
+
+      case EVENTID(BUTTON_AUX, EVENT_CLICK_LONG, MODE_ON):
+        swing_blast_ = true;
+        hybrid_font.SB_Blast();
+        return true;
+       
+       case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_ON):
+         if (swing_blast_) {
+           SaberBase::DoBlast();
+         }
+         return true;
+
+#if NUM_BUTTONS == 1
+      case EVENTID(BUTTON_NONE, EVENT_TWIST, MODE_ON | BUTTON_POWER):
+#endif
+#ifndef DISABLE_COLOR_CHANGE
+      case EVENTID(BUTTON_POWER, EVENT_CLICK_SHORT, MODE_ON | BUTTON_AUX):
+        ToggleColorChangeMode();
+        break;
+#endif
+
+        // Lockup
+      case EVENTID(BUTTON_NONE, EVENT_CLASH, MODE_ON | BUTTON_POWER):
+      case EVENTID(BUTTON_NONE, EVENT_CLASH, MODE_ON | BUTTON_AUX):
+        if (!SaberBase::Lockup()) {
+          if (pointing_down_) {
+            SaberBase::SetLockup(SaberBase::LOCKUP_DRAG);
+          } else {
+            SaberBase::SetLockup(SaberBase::LOCKUP_NORMAL);
+          }
+          swing_blast_ = false;
+          SaberBase::DoBeginLockup();
+          return true;
+        }
+        break;
+
+      case EVENTID(BUTTON_AUX, EVENT_CLICK_SHORT, MODE_ON | BUTTON_POWER):
+	SaberBase::SetLockup(SaberBase::LOCKUP_LIGHTNING_BLOCK);
+     swing_blast_ = false;
+	SaberBase::DoBeginLockup();
+	return true;
+
+      case EVENTID(BUTTON_NONE, EVENT_STAB, MODE_ON | BUTTON_POWER):
+      case EVENTID(BUTTON_NONE, EVENT_STAB, MODE_ON | BUTTON_AUX):
+        if (!SaberBase::Lockup()) {
+          SaberBase::SetLockup(SaberBase::LOCKUP_MELT);
+          swing_blast_ = false;
+          SaberBase::DoBeginLockup();
+          return true;
+        }
+        break;
+
+        // Off functions
+      case EVENTID(BUTTON_POWER, EVENT_CLICK_LONG, MODE_OFF):
+        StartOrStopTrack();
+        return true;
+
+      case EVENTID(BUTTON_POWER, EVENT_PRESSED, MODE_OFF):
+        SaberBase::RequestMotion();
+        return true;
+
+case EVENTID(BUTTON_AUX, EVENT_CLICK_LONG, MODE_OFF):
+    if (mode_volume_){
+      mode_volume_ = false;
+      hybrid_font.SB_Change(EXIT_COLOR_CHANGE);
+      // beeper.Beep(0.5, 3000);
+      STDOUT.println("Exit Volume Menu");
+    }
+    else{
+      mode_volume_ = true;
+      hybrid_font.SB_Change(ENTER_COLOR_CHANGE);
+      // beeper.Beep(0.5, 3000);
+      STDOUT.println("Enter Volume Menu");
+    }
+    return true;
+
+      case EVENTID(BUTTON_NONE, EVENT_CLASH, MODE_OFF | BUTTON_POWER):
+#if NUM_BUTTONS == 0
+      case EVENTID(BUTTON_NONE, EVENT_SHAKE, MODE_OFF):
+#endif
+        next_preset();
+        return true;
+
+      case EVENTID(BUTTON_POWER, EVENT_CLICK_SHORT, MODE_OFF | BUTTON_AUX):
+        previous_preset();
+        return true;
+
+      case EVENTID(BUTTON_AUX2, EVENT_CLICK_SHORT, MODE_OFF):
+#ifdef DUAL_POWER_BUTTONS
```

Comment:
This prop doesn't really support DUAL_POWER_BUTTONS, so I suggest just calling previous_preset() here